index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>My Portfolio</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+
+</head>
+
+<body>
+    <header class="container">
+        <nav class="nav-item1">My Logo
+        </nav>
+        <nav class="nav-item2">
+        </nav>
+        <nav class="nav-item3">
+            <img src="images/toggle.png" alt="toggle">
+        </nav>
+    </header>
+    <main class="content">
+            <h1 class="box1">I’m Emmanuel
+                Glad to see you!</h1>
+        <p class="box2">
+            I'm a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
```

Comment:
- [ ] By following the template design I found out that this 👆  paragraph at the center but your is still not yet centered. 
![image](https://user-images.githubusercontent.com/65068771/148311854-478f2fd7-f016-4858-b83e-af4199cb3507.png)